lib/matplotlib/texmanager.py

```diff
@@ -37,6 +37,23 @@
 _log = logging.getLogger(__name__)
 
 
+def _usepackage_if_not_loaded(package, option=None):
```

Comment:
```suggestion
def _usepackage_if_not_loaded(package, *, option=None):
```
It's not public, so not super important. But let's force ourselves to write readable code.

Reply:
Um, for some reason the suggestion doesn't work here (using the GitHub app). Proposal is to make `option` kwonly.

Reply:
sure